resources/qml/Menus/ConfigurationMenu/ConfigurationMenu.qml

```diff
@@ -38,16 +38,16 @@ Cura.ExpandablePopup
     headerItem: Item
     {
         // Horizontal list that shows the extruders and their materials
-        RowLayout
+        Row
         {
             anchors.fill: parent
             Repeater
             {
                 model: extrudersModel
                 delegate: Item
                 {
-                    Layout.fillWidth: true
```

Comment:
Layout.preferredWidth: Math.round(parent.width / extrudersModel.count) would probably do the trick as well if you wanna keep the RowLayout component (which is better for re-sizable interfaces).

Reply:
Thanks for the suggestion Chris. I converted it back to RowLayout